signup/signup.js

```diff
@@ -59,30 +63,72 @@ function pwcheckToggleIcon(event) {
   input.togglePasswordType(signupPwCheckInput, signupPwcheckIcon);
 }
 
-//엔터키
-function handlesignupButtonEnter(event) {
-  if (event.code === 'Enter') {
-    handlesignupButtonClick();
-  }
-}
+//버튼 part
+const signupButton = document.querySelector('.signup__form--signupBt');
+signupButton.addEventListener('click', handlesignupButtonClick);
 
 //로그인 버튼 클릭 시
-function handlesignupButtonClick(event) {
-  if(signupEmailInput.value !== '' && signupEmailInput.value !== '' && signupEmailInput.value !== '' ){
-    if (signupEmailInput.value !== 'test@codeit.com') {
-      if (signupPwInput.value === signupPwCheckInput.value) {
+async function handlesignupButtonClick(event) {
+  //input이 비어있을 때
+  if (
+    signupEmailInput.value === "" ||
+    signupPwInput.value === "" ||
+    signupPwCheckInput.value === ""
+  ) {
+    differentEmailPw();
+    return;
+  }
+  //비밀번호가 일치하지 않았을 때 
+  pwCheckHandleFocusOut(signupPwCheckInput, signupPwInput, signupPwCheckError);
+  if (signupPwInput.value !== signupPwCheckInput.value) {
+    return;
+  }
```

Comment:
어짜피 동작은 똑같을테지만 이 둘의 순서가 바뀌어야 논리적으로 이해하기 쉬울것 같네요. 
```suggestion
  if (signupPwInput.value !== signupPwCheckInput.value) {
    pwCheckHandleFocusOut(signupPwCheckInput, signupPwInput, signupPwCheckError);
    return;
  }
```